src/api/app.py

```
@@ -26,12 +35,52 @@
+        token = request.form.get('_csrf_token')
+    
+    stored_token = session.get('_csrf_token')
+    if not token or not stored_token or token != stored_token:
+        abort(400, description="CSRF token missing or invalid")
+
+# Custom request parser for JSON requests
+@app.before_request
+def parse_json_request():
+    if request.is_json:
+        try:
+            request.json
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)